src/simulation/thermal/motorThermal/_UnitTest/test_motorThermal.py

```diff
@@ -218,16 +218,6 @@ def motorThermalTest(show_plots, accuracy):
     unitTestSim.ExecuteSimulation()
     numTests += 1
 
-    # change the test conditions
```

Comment:
I don't see you testing the ability to change the RW data on the fly?  We should have a unit test that verifies this behavior works.